src/main/webapp/search-interview.js

```diff
@@ -18,42 +18,67 @@ function onSearchInterviewLoad() {
+    .then(response => response.text())
+      .then(interviewTimes => {
+        interviewTimesDiv().innerHTML = interviewTimes;
+      });
+}
+
+function interviewTimesDiv() {
+  return document.getElementById('interview-times-container');
+}
+
+function browserTimezoneOffset() {
+  let date = new Date();
+  return (-1) * date.getTimezoneOffset();
 }
 
 // Confirms interview selection with user and sends this selection to Datastore 
 // if confirmed.
 function selectInterview(interviewer) {
+  let date = interviewer.getAttribute('data-date');
+  let time = interviewer.getAttribute('data-time');
+  let company = interviewer.getAttribute('data-company');
+  let job = interviewer.getAttribute('data-job');
+  let utcStartTime = interviewer.getAttribute('data-utc');
   if (confirm(
-    // TOOD: fill in these times dynamically from ids in the .jsp file.
-      `You selected: Sunday 7/5 from 6:30 PM - 7:30 PM with a ` +
-      `${interviewer.getAttribute("data-company")} ` +
-      `${interviewer.getAttribute('data-job')}. ` +
+      `You selected: ${date} from ${time} with a ` +
+      `${company} ${job}. ` +
       `Click OK if you wish to proceed.`)) {
     alert(
-      `You have scheduled an interview on Sunday 7/5 from 6:30 PM - 7:30 ` +
-      `PM with a ${interviewer.getAttribute('data-company')} ` +
-      `${interviewer.getAttribute('data-job')}. Check your email for more ` +
+      `You have scheduled an interview on ${date}` +
+      ` from ${time} with a ${company} ` +
+      `${job}. Check your email for more ` +
       `information.`);
-    // TODO: Call a servlet to save this selection.
-    location.reload();
+    let requestObject = {
+      company: company,
+      job: job,
+      utcStartTime: utcStartTime
+    };
+    let requestBody = JSON.stringify(requestObject);
+    let request = new Request('/scheduled-interviews', {method: 'POST', body: requestBody});
+    fetch(request).then(unused => {window.location.replace('/scheduled-interviews.html');});
   }
 }
 
 // Fills in the modal with interviewer info from Datastore and shows it.
 function showInterviewers(selectButton) {
-  fetch('/search-interview-interviewers.jsp').then(response => response.text())
-    .then(table => {
-      $('#modal-body').html(table);
-      // TODO: format the date as June 5, 2020 not 7/5/2020 (more readable).
-      const date = selectButton.getAttribute("id");
-      const time = document.getElementById(date + '-options').value;
-      $('#modal-title').text(`Interviewers Information for ${date} at ${time}`);
-    });
+  const date = selectButton.getAttribute('data-date');
+  const time = document.getElementById(date).innerText;
+  const reformattedTime = time.replace('-', 'to');
+  const utc = document.getElementById(date).value;
+  fetch(`/show-interviewers?utcStartTime=${utc}&date=${date}&time=${reformattedTime}`)
+    .then(response => response.text())
+      .then(interviewers => {
```

Comment:
fix indentation.

(why is the indentation on all of these messed up?)

Reply:
Done